### ISSUE TITLE
Malformed SLD when uploading GeoJSON

### ISSUE THREAD

**Issue by fzadrazil** [hslayers/hslayers-ng#3031]
# Bug

**Describe the bug**
When publishing GeoJSON with polygon features, malformed style is created for WFS layer. The layer seems to have default symbolizer rules for all the possible geometry types even if there is just polygons in the layer. No features are displayed in the map as a result.

![image](https://user-images.githubusercontent.com/48727502/168844781-3731ae5c-c17b-404d-83f6-22f5a853f17f.png)

**To Reproduce**
1. Go to the Add data panel and upload a geojson file with polygon features 
[Kraje_simple.zip](https://github.com/hslayers/hslayers-ng/files/8709442/Kraje_simple.zip)
Now the layer is displayed in the map as expected.
2. Remove the added layer from the map
3. Add the same layer as WFS from the catalogue
4. The layer is not visible in the map
5. Open the symbolizer and see the default style rules set

**Expected behavior**
The WFS should keep the original symbology.

**Additional context**
Opening the same layer in QGIS Layman plugin shows the same effect, rule-based symbology is applied to the layer with four default symbols (point, line, polygon, raster). Publishing the very same layer from QGIS works fine, the symbology works as expected.

**Comment by raitisbe** [comment]
Not sure its a bug, because default ol style has these symbols. Dont know
where the raster comes from

**Comment by fzadrazil** [comment]
Yes, I can understand that those symbols are there, but the raster is strange and the result surely seems like a bug, because the layer is not rendering any features. It does not even help to remove those other styles or change attributes of the polygon style, still not visible until you reset the styling completely.
![image](https://user-images.githubusercontent.com/48727502/168854242-875194df-38d5-4e28-bfd0-0d9be0a87912.png)

**Comment by FilipLeitner** [comment]
Wasnt able to debug on hsl side yet but found 'raster' named sld rule in layman repo https://github.com/LayerManager/layman/blob/11f7c0400b09b1e4c6ddea0f053d0395d789fe5b/deps/geoserver/sample/geoserver_data/styles/default_raster.sld

**Comment by raitisbe** [comment]
Whats under rule filters? I remember having some strange filter expressions coming for from qgis for some wfs. Mayube its also the case here?
